### DISCUSSION TITLE
associations() got an unexpected keyword argument 'bias_correction'

### DISCUSSION THREAD

**Discussion by DupploDev** [shakedzy/dython#126] [Q&A] [answered]
I'm a user of Dython and associations package, wich has beeing very useful to develop Scorecards using correlation matrixes, but since I've changed my laptop and also install Python, and also install all the packaged, I'm unable to use this to my new data because I got the same error and I'm breaking my head and Can't find any solution, Please Help!!

corr_matrix = associations(df5, nominal_columns="auto",
                           numerical_columns=None,
                           mark_columns=False,
                           nom_nom_assoc="cramer",
                           num_num_assoc="pearson",
                           bias_correction = 'True',
                           nan_strategy=False,
                           nan_replace_value=0.0,
                           ax=None, figsize=None, annot=True,
                           fmt=".2f", cmap=None, sv_color="silver",
                           cbar=True, vmax=1.0, vmin=None,
                           plot=True, compute_only=False, 
                           clustering=False, title=None, filename=None)

associations() got an unexpected keyword argument 'bias_correction'

**Comment by shakedzy** [comment, marked as answer]
Hey @DupploDev - that keyword was renamed to `cramers_v_bias_correction` in version 0.7.0, but for some reason I forgot to update the online documentation with that change.. I'll do that soon. In the meantime, you can use the new keyword just as you used the old one. Thanks for paying attention to this!
